SwiftWisdom/Core/NSFormatter/DecimalFormatter.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  DecimalFormatter.swift
+//  SwiftWisdom
+//
+//  Created by Stephen Wong on 5/1/16.
+//  Copyright © 2016 Intrepid. All rights reserved.
+//
+
+import Foundation
+
+final class DecimalFormatter: NSNumberFormatter {
```

Comment:
Also, I suggest making this private to the file that uses it, unless we feel there is a compelling reason to expose it.